src/Utilities/Tree.hh

```diff
@@ -88,7 +88,7 @@ public:
 
   //--------------------------- Private Interface ----------------------------//
   // Define the types we use to build the tree.
-  typedef boost::unordered_map<CellKey, Cell> TreeLevel;
+  typedef std::unordered_map<CellKey, Cell> TreeLevel;
```

Comment:
@mdavis36 @jmikeowen just a note, there are a few changes like this which Peter Robinson pushed last week to get Spheral to build with nvcc. Apparently it did not like the v1.74.0 implementation of boost::unordered_map.

Reply:
@ptsuji We have been building Spheral with boost v1.74.0 and NVCC 11.1.1 for quite some time now. I'm not opposed personally to moving to the `std` implementation but I would be curious to know why that is failing for you and not us. What Cuda version are you building with?

Reply:
We're building with cuda 11.8.0 (compiler is /usr/tce/packages/cuda/cuda-11.8.0/bin/nvcc). @robinson96 can you comment on what the issue with nvcc was?

Reply:
My best recollection is that the std lib for the version of gcc we were on did not have hash functions defined for some of the key types spheral was using.